build.xml

```diff
@@ -50,7 +50,8 @@
             <arg path="src"/>
             <arg value="xml"/>
             <arg value="phpmd.xml"/>
-            <arg value="--exclude */Tests/*"/>
+            <arg value="--exclude"/>
```

Comment:
Ah, `line` instead of `value` was the issue. Well, we could fix this another time. It has been done this way in aforementioned repositories, so let's keep it the same.